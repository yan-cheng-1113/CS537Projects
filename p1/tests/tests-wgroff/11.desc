Improper format (invalid section number) - should exit returning 0 and print  usage message
