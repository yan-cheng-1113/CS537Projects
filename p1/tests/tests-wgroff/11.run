cp /p/course/cs537-oliphant/tests/P1/tests-wgroff/bad_header_section.txt .; ./wgroff bad_header_section.txt
