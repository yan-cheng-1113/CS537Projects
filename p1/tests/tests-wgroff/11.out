Improper formatting on line 1
